src/providers/oracle-authorization.provider.ts

```diff
@@ -44,7 +45,22 @@ export class OracleAuthorizationProvider implements Provider<Authorizer> {
               return AuthorizationDecision.DENY;
           },
         };
-      return rules[request.method]() ?? AuthorizationDecision.ALLOW;
+      return rules[request.method]() ?? this.DEFAULT_AUTH_DECISION;
+    },
+    auth: ({
+      request,
+      oracle,
+    }: AuthorizationCheckParams): AuthorizationDecision => {
+      const rules: {[method: string]: () => AuthorizationDecision | undefined} =
+        {
+          POST: () => {
+            // only allow SYSTEM account
+            return oracle.id === 1
```

Comment:
We've dragged this a bit everywhere. Let's have this magic number moved to a constant that we import from shared file.
```suggestion
            return oracle.id === SYSTEM_ID
```

Reply:
see 9f7bd0a381a326c5dab774f2f7fa665aa337e4b3

Reply:
I can change it to just SYSTEM_ID instead of SYSTEM_ORACLE_ID, but that is just semantics